sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/pom.xml

```diff
@@ -17,7 +17,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
-  <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;current} -->
+  <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;current} -->
```

Comment:
This has a dependency on azure-storage-blob, and they are doing a `12.9.0` in Nov month, will eventhubs packages be shipping again in that case?

Reply:
We will be shipping EH again when next version of AMQP is ready for GA in mid-Nov. This will align with SB GA release.